tdex/docker-compose.yml

```diff
@@ -7,7 +7,7 @@ services:
       APP_PORT: 8080
   
   tdexd:
-    image: ghcr.io/tdex-network/tdexd:v0.8.19@sha256:5a2c0db9f74812e67aa98f50f396e81edd23ef49cb6797b084027ba6124db707
+    image: ghcr.io/tdex-network/tdexd:v0.0.9@sha256:77729e6c0019752059cc5bf624675e261abd8b856e0a7e8f59889b7ec0a35eb3
```

Comment:
@tiero Is there a typo here as I can't pull down `v0.0.9`, but I see `v0.9.0` available?

Reply:
Ups, yes typo before commit